src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -730,6 +719,11 @@
     <Compile Include="System\Xml\Xsl\Xslt\XslVisitor.cs" />
     <Compile Include="$(CommonPath)\System\LocalAppContext.cs" />
     <Compile Include="System\Xml\Core\LocalAppContextSwitches.cs" />
+    <Compile Include="System\CodeDom\CodeDomProvider.cs" />
+    <Compile Include="System\CodeDom\CodeGenerator.cs" />
+    <Compile Include="System\CodeDom\CompilerError.cs" />
+    <Compile Include="System\CodeDom\CompilerResults.cs" />
+    <Compile Include="System\CodeDom\TempFileCollection.cs" />
```

Comment:
I'm confused what these are for if you're ripping out CodeDOM dependency (or putting behind #if CODEDOM everywhere)